src/components/app.tsx

```diff
@@ -133,45 +133,29 @@ const App: FunctionComponent = () => {
         })
     );
 
-    const initialize: () => Promise<UserManager> = useCallback(() => {
-        if (process.env.REACT_APP_USE_AUTHENTICATION === 'true') {
-            return fetchAuthorizationCodeFlowFeatureFlag().then(
-                (authorizationCodeFlowEnabled) =>
-                    initializeAuthenticationProd(
-                        dispatch,
-                        initialMatchSilentRenewCallbackUrl != null,
-                        fetch('idpSettings.json'),
-                        fetchValidateUser,
-                        authorizationCodeFlowEnabled
-                    )
-            );
-        } else {
-            return initializeAuthenticationDev(
-                dispatch,
-                initialMatchSilentRenewCallbackUrl != null,
-                () =>
-                    new Promise((resolve) =>
-                        window.setTimeout(() => resolve(true), 500)
-                    )
-            );
-        }
-        // Note: initialMatchSilentRenewCallbackUrl and dispatch don't change
-    }, [initialMatchSilentRenewCallbackUrl, dispatch]);
-
     useEffect(() => {
-        initialize()
+        AppsMetadataSrv.fetchAuthorizationCodeFlowFeatureFlag()
+            .then((authorizationCodeFlowEnabled) =>
+                initializeAuthenticationProd(
+                    dispatch,
+                    initialMatchSilentRenewCallbackUrl != null,
+                    fetch('idpSettings.json'),
+                    UserAdminSrv.fetchValidateUser,
+                    authorizationCodeFlowEnabled
+                )
```

Comment:
You have to add a new flag on signin-callback URL, see PR #3

Reply:
fixed with update branch from main